ui/src/pages/Upload.tsx

```diff
@@ -112,149 +176,206 @@ export const Upload: React.FC = () => {
+              placeholder="Lorem ipsum"
+            />
+            <UploadLabelsText>Last Name</UploadLabelsText>
+            <StyledTextField
+              onChange={handleChange}
+              variant="outlined"
+              required
+              id="author-last-name"
+              name="author_last_name"
+              label="Enter author's last name"
+              placeholder="Lorem ipsum"
+            />
+            <UploadLabelsText>Biography</UploadLabelsText>
+            <StyledTextField
+              variant="outlined"
+              onChange={handleChange}
+              multiline
+              placeholder="Enter additional information here"
+              rows={4}
+              id="author-bio"
+              name="bio"
+              label="Enter author biography"
+            />
+          </StyledBackgroundColor>
 
-          <FormControl>
-            <InputLabel id="Current Location">Current City</InputLabel>
-            <Select
+          <StyledBackgroundColor>
+            <UploadStoriesHeading>Additional Information</UploadStoriesHeading>
+            <FormControl>
+              <UploadLabelsText>Country of Origin</UploadLabelsText>
+              <StyledInputLabel id="Country of Origin">
+                Enter story&#39;s country of origin
+              </StyledInputLabel>
+              <StyledSelect
+                variant="outlined"
+                value={formInput.author_country}
+                onChange={handleChange}
+                inputProps={{
+                  name: "author_country",
+                  id: "select-label-country",
+                }}
+              >
+                {countriesList.map((country) => (
+                  <StyledMenuItem key={country.code} value={country.name}>
+                    {country.name}
+                  </StyledMenuItem>
+                ))}
+              </StyledSelect>
+            </FormControl>
+            <UploadLabelsText>Current City</UploadLabelsText>
+            <StyledInputLabel id="Current Location">
+              Enter where story was written
+            </StyledInputLabel>
+            <StyledSelect
+              variant="outlined"
               value={formInput.current_city}
               onChange={handleChange}
               inputProps={{
                 name: "current_city",
                 id: "select-label-city",
               }}
             >
-              <MenuItem value={"Toronto"}>Toronto</MenuItem>
-              <MenuItem value={"Calgary"}>Calgary</MenuItem>
-              <MenuItem value={"Vancouver"}>Vancouver</MenuItem>
-              <MenuItem value={"Thunder Bay"}>Thunder Bay</MenuItem>
-            </Select>
-          </FormControl>
-
-          <FormControl>
-            <InputLabel id="input-label-year">Year Published</InputLabel>
-            <Select
-              value={formInput.year}
+              <StyledMenuItem value={"Toronto"}>Toronto</StyledMenuItem>
```

Comment:
Should we have a TODO to pull example city names from the backend? Can be a low priority task.

Reply:
rather than backend, would it make sense to have a list of all Canadian cities like we do for countries. Might be a overkill and not needed tho

Reply:
I'll add in the cities Jenna's going to send me and then add a low priority ticket to be able to add more cities in the future.